extensions/ui/org.eclipse.smarthome.ui.basic/snippets/switch.html

```diff
@@ -2,13 +2,17 @@
 	<span class="mdl-form__icon">
 		<img data-icon="%category%" src="../icon/%category%?state=%state%&format=%icon_type%" />
 	</span>
-	<div class="mdl-form__label">
+	<span %labelstyle% class="mdl-form__label">
 		%label%
-	</div>
+	</span>
+	<span %valuestyle% class="mdl-form__value mdl-form__value--group">
```

Comment:
`mdl-form__value--group` should be `mdl-form__value--switch`

And the value needs some padding:

```less
// inside .mdl-form__value
&--switch {
	padding: 0 10px 0 10px;
}
```